quickcheck/tests/test.hs

```diff
@@ -107,6 +109,9 @@ main =
           resultDescription =~ "Failed.*expected failure"
           resultDescription =~ "Use .* to reproduce"
 
+      -- ???????? not sure how to test intermediate results
```

Comment:
```suggestion
      -- Run the test suite manually and check that progress does not go beyond 100%
```

Reply:
@MaximilianAlgehed Do you have any ideas on how to test this properly?

Reply:
We don't really have UI tests of this sort, so let's not overthink it much.